apps/total-typescript/src/video/module-progress.ts

```diff
@@ -18,23 +18,25 @@ export const ModuleProgressSchema = z.object({
       slug: z.string(),
     })
     .nullish(),
-  sections: z.array(
-    z.object({
-      id: z.string(),
-      slug: z.string(),
-      sectionCompleted: z.boolean(),
-      percentComplete: z.number(),
-      lessonCount: z.number(),
-      completedLessonCount: z.number(),
-      lessons: z.array(
-        z.object({
-          id: z.string(),
-          slug: z.string(),
-          lessonCompleted: z.boolean().default(false),
-        }),
-      ),
-    }),
-  ),
+  sections: z
+    .array(
+      z.object({
+        id: z.string(),
+        slug: z.string(),
+        sectionCompleted: z.boolean(),
+        percentComplete: z.number(),
+        lessonCount: z.number(),
+        completedLessonCount: z.number(),
+        lessons: z.array(
+          z.object({
+            id: z.string(),
+            slug: z.string(),
+            lessonCompleted: z.boolean().default(false),
+          }),
+        ),
+      }),
+    )
+    .nullable(),
```

Comment:
this is where I'd like this to be "always an array" so a module **always** has sections